extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/handler/CircuitHandler.java

```diff
@@ -108,14 +109,14 @@ private synchronized BridgeHandler getDssBridgeHandler() {
 
             if (handler instanceof BridgeHandler) {
                 dssBridgeHandler = (BridgeHandler) handler;
-                dssBridgeHandler.registerDeviceStatusListener(this);
+                // dssBridgeHandler.registerDeviceStatusListener(this);
             } else {
                 return null;
             }
         }
-        if (circuit == null) {
-            dssBridgeHandler.registerDeviceStatusListener(this);
-        }
+        // if (circuit == null) {
```

Comment:
maybe its better to fully remove these lines instead of just commenting them out.